site_config_client/openedx/features.py

```diff
@@ -15,6 +15,21 @@ def is_feature_enabled():
     return enabled_globally
 
 
+def enable_feature_for_site(site_uuid, note=''):
+    """
+    Enable the site configuration client for a site by site_uuid.
+
+    Returns: whether a new flag is created or not.
```

Comment:
`get_or_create()` returns a tuple of the created (or gotten) object and the boolean flag. `enable_feature_for_site()` is just passing along its results, so really that's what it's returning as well. We should probably either be clear about that in the description or change the code below to `_, created = SiteConfigClientEnabled.objects.get_or_create(...`. so that we're only returning the flag.

Reply:
good catch, I've forgot to refactor it